<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditClientWidget</class>
 <widget class="QWidget" name="AuditClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audit Client</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Connexion au serveur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse IP du serveur :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Num√©ro de port du serveur :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Etat de la connexion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Information du client :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAdresseIP">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPort">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuitter">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnexion">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Connexion</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditEtat">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>431</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOrdinateur">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de l'odinateur distant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUtilisateur">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de l'utilisateur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonArchitecture">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Architecture processeur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OS de l'ordinateur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOrdinateur">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>721</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUtilisateur">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>721</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditArchitecture">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>721</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOS">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>721</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>AuditClientWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
